### PR TITLE
Update RF24registers.h

### PR DESCRIPTION
**Pull request by Berkseo**
https://forum.mysensors.org/topic/9598/smartsleep-wake-up-message-always-get-nack/11

### CODE REVIEW

**Comment by mfalkvidd** [comment]
Could the problem be realted to https://forum.mysensors.org/topic/9642/nrf24-transmission-of-data-works-fine-but-constant-nack-s-produced/ so the fix suggested there is a better solution?

**Comment by Berkseo** [comment]
Increasing the number of attempts is aimed at correcting this line - 2912 !TSF:MSG:SEND,100-100-0-0,s=255,c=3,t=26,pt=1,l=1,sg=0,ft=0,st=NACK:2.

This does not apply to NOACK in the first message after waking up

As I understand in this the frost is offered to exclude delay in 10ms(RF24.cpp). I think it's the wrong decision. The specification says it explicitly - at least 10 MS (>=10). For the original radio modules, a 10ms delay is probably sufficient to output the signal to the output. For most radio modules, this is clearly not enough.

File: drivers/RF24/RF24registers.h [hunk 1]
```diff
@@ -41,7 +41,7 @@
 #define RF24_CRC_16				(3)
 
 // ARD, auto retry delay
-#define RF24_SET_ARD			(5)	//=1500us
+#define RF24_SET_ARD			(10)	//=1500us
```
Comment (mfalkvidd) [inline drivers/RF24/RF24registers.h]: Is the comment (1500us) still valid after this change?

**Comment by Berkseo** [comment]
no, I'm sorry, I just did not remove the comment, of course there will be more time.

**Comment by Yveaux** [comment]
This solution is likely to conflict with #1134.

**Comment by tekka007** [comment]
see #1193

**Pull request closed without merging**